remoteappmanager/handlers/home_handler.py

```
@@ -125,11 +124,21 @@ def _actionhandler_view(self, options):
         It is not different from pasting the appropriate URL in the
         web browser, but we validate the container id first.
         """
-        container = yield self._container_from_options(options)
+        try:
+            url_id = options["url_id"][0]
+        except (KeyError, IndexError):
+            message = "Failed to retrieve valid url_id from form."
+            self.log.exception(message)
+            self.finish(message)
```

Same here, let it raise.